app/src/pages/trace/EditSpanAnnotationsDialog.tsx

```diff
@@ -325,7 +332,11 @@ function SpanAnnotationCard(props: {
           ]
         ) {
           query {
-            node(id: $spanId) {
+            project: node(id: $projectId) {
+              ...AnnotationSummaryValueFragment
```

Comment:
this only needs to refetch the summary value fragment since this annotation name must have already existed and will still exist after edit

Reply:
I'm gonna nix the cache busting for now - it will break the experiments UI